### PR TITLE
Upgrade: Add autoscaler test

### PR DESCRIPTION
**Pull request by jhou1**
Adding one more test for autoscaler test. @sunzhaohua2 @miyadav PTAL

Tested and passed.
- prepare: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/99284/
- check: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/99286/

### CODE REVIEW

File: features/upgrade/machines/machines.feature [hunk 1]
```diff
@@ -75,3 +75,62 @@ Feature: Machine-api components upgrade tests
...
     And the machineset should have expected number of running machines
+
+  @upgrade-prepare
+  @admin
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed - prepare
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+    When I run the :create admin command with:
+      | f | <%= BushSlicer::HOME %>/testdata/cloud/cluster-autoscaler.yml |
```
Comment (akostadinov) [inline features/upgrade/machines/machines.feature:86]: please use step
```
I obtain test data file "cloud/cluster-autoscaler.yml"
```
then use the file.

File: features/upgrade/machines/machines.feature [hunk 2]
```diff
@@ -75,3 +75,62 @@ Feature: Machine-api components upgrade tests
...
+    And I use the "openshift-machine-api" project
+    And admin ensures machine number is restored after scenario
+
+    Given I clone a machineset and name it "machineset-clone-28108"
+
+    # Delete clusterautoscaler after scenario
+    Given admin ensures "default" clusterautoscaler is deleted after scenario
+
+    # Create machineautoscaler
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/cloud/machine-autoscaler.yml" replacing paths:
```
Comment (akostadinov) [inline features/upgrade/machines/machines.feature:108]: same here

File: features/upgrade/machines/machines.feature (hunk 1 above)
Comment (akostadinov) [reply features/upgrade/machines/machines.feature:86]: and all other places where `BushSlicer::HOME`

Comment (jhou1) [reply features/upgrade/machines/machines.feature]: Updated, thanks

**Comment by jhou1** [comment]
Tested and passed https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/99555/console

@sunzhaohua2 @miyadav PTAL.

File: features/upgrade/machines/machines.feature [hunk 3]
```diff
@@ -75,3 +75,66 @@ Feature: Machine-api components upgrade tests
...
+  @upgrade-check
+  @admin
+  @destructive
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+    And admin ensures machine number is restored after scenario
+
+    Given I clone a machineset and name it "machineset-clone-28108"
```
Comment (liangxia) [inline features/upgrade/machines/machines.feature:104]: What's the meaning of 28108 ?

Comment (jhou1) [reply features/upgrade/machines/machines.feature]: The case id, apparently it should be 30783 here. Updated, thanks

**Pull request merged**